ntoskrnl/se/token.c

```diff
@@ -363,6 +363,155 @@ SepCompareTokens(
     return STATUS_SUCCESS;
 }
 
+/**
+ * @brief
+ * Determines if a system anonymous logon token should be returned
+ * including the Everyone Group SID (SeWorldSid) or without the
+ * said group by querying the EveryoneIncludesAnonymous registry value.
+ *
+ * @param[out] IncludeEveryone
+ * The retrieved value indicating if the anonymous logon token should
+ * be returned including the Everyone Group SID. If the value is 0 (FALSE)
+ * then a token will be returned without this group.
+ *
+ * @return
+ * Returns STATUS_SUCCESS if the registry data value querying has completed
+ * successfully, a failure NTSTATUS code is returned otherwise.
+ */
+static
+NTSTATUS
+SepIncludeEveryoneGroupAnonymous(
+    _Out_ PBOOLEAN IncludeEveryone)
+{
+    NTSTATUS Status;
+    ULONG IncludeEveryoneValueData;
+    static UNICODE_STRING LsaKey = RTL_CONSTANT_STRING(L"\\Registry\\Machine\\SYSTEM\\CurrentControlSet\\Control\\Lsa");
+    static UNICODE_STRING IncludeEveryoneValue = RTL_CONSTANT_STRING(L"EveryoneIncludesAnonymous");
```

Comment:
You seem to use only `Buffer` fields from these. Why do you need a `UNICODE_STRING` then?